index.bs

```diff
@@ -189,66 +195,76 @@ For multi-layered images, the LayerSelectorProperty may be present as defined in
 
 <h3 id="image-and-image-collection-brand">AVIF image and image collection brand</h3>
-Files that conform with the profile-independent restrictions in this document (sections [[#image-item-and-properties]] and [[#auxiliary-images]]) should include in the [=compatible_brands=] field of the [=FileTypeBox=]:
-- the brand <dfn value="" export="" for="AVIF Image brand">avif</dfn> ,
-- the brand 'mif1' as recommended in [[!HEIF]],
-- a brand to identify the AVIF profile (see section [[#profiles]]), if any, with which the file complies.
+The brand to identify [=AV1 image items=] is <dfn value="avif" export="" for="AVIF Image brand">avif</dfn>.
+
+Files that indicate this brand shall comply with the following:
+- there shall be an [=AV1 image item=] among the set of items comprised of the primary item and its alternates.
+- [=AV1 auxiliary image items=] as specified in [[#auxiliary-images]] may be present in the file.
+
+Additionally, the brand <dfn value="avio" export="" for="AVIF Intra-only brand">avio</dfn> is defined. If the file indicates the brand <code>[=avio=]</code> in the [=compatible_brands=] field of the [=FileTypeBox=], then there shall be an [=AV1 image item=] made only of [=Key Frames=] among the set of items comprised of the primary item and its alternates. Conversely, if such item exists, the brand <code>[=avio=]</code> should be used.
 
 <h3 id="image-sequence-brand">AVIF image sequence brands</h3>
-Files which contain one or more image sequences, and which conform with the profile-independent restrictions in this document (sections [[#image-item-and-properties]], [[#image-sequences]] and [[#auxiliary-images]]), should include in the [=compatible_brands=] field of the [=FileTypeBox=]:
-- the brand <dfn value="" export="" for="AVIF Image Sequence brand">avis</dfn>,
-- the brand 'msf1' as recommended in [[!HEIF]],
-- a brand to identify the AVIF profile (see section [[#profiles]]), if any, with which the file complies.
+The brand to identify AVIF image sequences is <dfn value="avis" export="" for="AVIF Image Sequence brand">avis</dfn>.
+
+Files that indicate this brand shall comply with the following:
+- they shall contain one or more [=AV1 image sequences=] as defined in [[#image-sequences]].
+- they may contain [=AV1 auxiliary image sequences=] as defined in [[#auxiliary-images]].
 
-Additionally, if the image sequences are made only of [=AV1 Samples=] marked as 'sync', then the brand <dfn value="" export="" for="AVIF Intra-only brand">avio</dfn> should be used.
+Files that conform with these constraints should include in the [=compatible_brands=] field of the [=FileTypeBox=]:
+- the brand <code>'[=avis=]'</code>,
```

Comment:
Nit/optional: If the major_brand field is "avis", it is not clear whether this sentence implies that "avis" should appear in the compatible_brands field again.

Reply:
It is a common practice to replicate the major brand in the compatible brands, but I will double check the impact of not doing it.